azure-kusto-data/azure/kusto/data/helpers.py

```diff
@@ -1,10 +1,30 @@
+from __future__ import annotations
```

Comment:
Don't use this - it breaks libraries. Just you quote your annotations ("pd.Timedelta")